optimum/neuron/distributed/checkpointing.py

```diff
@@ -38,13 +38,11 @@ def consolidate_tensor_parallel_checkpoints(checkpoint_dir: Union[str, Path]) ->
 
     state_dicts = []
 
-    for sharded_checkpoint in checkpoint_dir.glob("tp_rank_*"):
+    for sharded_checkpoint in sorted(checkpoint_dir.glob("tp_rank_*/checkpoint.pt")):
```

Comment:
This will only work for TP right?
It will need to be updated for TP + PP. That's not an issue, just checking.